src/lib/infusion/Infusion-LICENSE.txt

```diff
@@ -0,0 +1,227 @@
+Infusion is available under either the terms of the New BSD license or the
```

Comment:
Unclear why these files from the Infusion distribution are copied here - this material should be sourced from npm